state/workingset.go

```diff
@@ -55,7 +55,8 @@ type (
 		cachedAccount    map[hash.PKHash]*State   // accounts being modified in this block
 		cachedContract   map[hash.PKHash]Contract // contracts being modified in this block
 		accountTrie      trie.Trie                // global state trie
-		dao              db.CachedKVStore         // the underlying DB for account/contract storage
+		cb               db.CachedBatch           // cached batch for pending writes
```

Comment:
it seems that we never use cb in workingset. Could we move the initialization of cb into where it is used, say trie?

Reply:
thx for careful review
the reason is that if the block contains contract, each contract will start a new trie, all the contracts must use the same cb
so init a new cb at beginning, and pass it to contract creation in getContract():

tr, err := trie.NewTrieSharedBatch(ws.dao, ws.cb, trie.ContractKVNameSpace, state.Root)